splink/duckdb/duckdb_comparison_template_library.py

```diff
@@ -0,0 +1,9 @@
+from ..comparison_template_library import DateComparisonBase
+from .duckdb_comparison_level_library import levenshtein_level
+from .duckdb_comparison_library import DuckDBComparisonProperties
+
+
+class date_comparison(DuckDBComparisonProperties, DateComparisonBase):
+    @property
+    def _levenshtein_level(self):
+        return levenshtein_level
```

Comment:
As mentioned on slack - I wonder if there's an elegant way of removing this.

I'll have a think.

Reply:
Not sure if there is a better way of doing things altogether (and haven't really looked through the meat of the PR yet), but if this is defined on `DuckDBComparisonProperties` instead (and sim. for other dialects), then it would only have to be done _once_, and then could be re-used in future template comparisons similarly (and would be more consistent with how this works for other comparisons)